packages/@aws-cdk/toolkit-lib/test/actions/diff.test.ts

```diff
@@ -54,38 +53,49 @@ describe('diff', () => {
     }));
   });
 
-  // TODO: uncomment when diff returns a value
-  // test('returns diff', async () => {
-  //   // WHEN
-  //   const cx = await builderFixture(toolkit, 'stack-with-bucket');
-  //   const result = await toolkit.diff(cx, {
-  //     stacks: { strategy: StackSelectionStrategy.ALL_STACKS },
-  //   });
+  test('returns diff', async () => {
+    // WHEN
+    const cx = await builderFixture(toolkit, 'stack-with-bucket');
+    const result = await toolkit.diff(cx, {
+      stacks: { strategy: StackSelectionStrategy.ALL_STACKS },
+    });
+
+    // THEN
+    expect(result['Stack1']).toMatchObject(expect.objectContaining({
+      resources: {
+        diffs: expect.objectContaining({
+          MyBucketF68F3FF0: expect.objectContaining({
+            isAddition: true,
+            isRemoval: false,
+            oldValue: undefined,
+            newValue: {
+              Type: 'AWS::S3::Bucket',
+              UpdateReplacePolicy: 'Retain',
+              DeletionPolicy: 'Retain',
+              Metadata: { 'aws:cdk:path': 'Stack1/MyBucket/Resource' },
+            },
+          }),
+        }),
+      },
+    }));
+  });
+
+  test('returns multiple template diffs', async () => {
+    // WHEN
+    const cx = await builderFixture(toolkit, 'two-empty-stacks');
+    const result = await toolkit.diff(cx, {
+      stacks: { strategy: StackSelectionStrategy.ALL_STACKS },
+    });
 
-  //   // THEN
-  //   expect(result).toMatchObject(expect.objectContaining({
-  //     resources: {
-  //       diffs: expect.objectContaining({
-  //         MyBucketF68F3FF0: expect.objectContaining({
-  //           isAddition: true,
-  //           isRemoval: false,
-  //           oldValue: undefined,
-  //           newValue: {
-  //             Type: 'AWS::S3::Bucket',
-  //             UpdateReplacePolicy: 'Retain',
-  //             DeletionPolicy: 'Retain',
-  //             Metadata: { 'aws:cdk:path': 'Stack1/MyBucket/Resource' },
-  //           },
-  //         }),
-  //       }),
-  //     },
-  //   }));
-  // });
+    // THEN
+    expect(result['Stack1']).toBeDefined();
+    expect(result['Stack2']).toBeDefined();
+  });
```

Comment:
Would like to see more thorough testing for multiple template diffs, i.e. more than just that the result is defined.